diff --git a/views_load_more.js b/views_load_more.js
index 15c03c4..ca22255 100644
--- a/views_load_more.js
+++ b/views_load_more.js
@@ -58,7 +58,7 @@
 
     // Add the new content to the page.
     wrapper.find('.pager a').remove();
-    wrapper.find('.pager').parent('.item-list').html(new_content.find('.pager'));
+    wrapper.find('.pager').replaceWith(new_content.find('.pager'));
     wrapper.find(content_query)[method](new_content.find(content_query).children());
 
     // Re-class the loaded content.
